docs/user-guide/leverage-cli/reference/basic-features.md

```diff
@@ -39,5 +41,5 @@ Tasks in build file `build.py`:
   start_server  [Default]	Start the server
   stop_server            	
 
-Powered by Leverage 1.0.10
+Powered by Leverage 1.9.0
```

Comment:
Just wondering if there is any way we can grab the version from single source of truth and place it anywhere it's needed. Just thinking out loud, not urgent at all.